lib/export/formatters/dataElementFormatterCimpl6.js

```diff
@@ -320,7 +317,9 @@ class DataElementFormatterCimpl6 {
             }
             else {
               if (fClone.constraints.length > 0) {
-                formattedFields.push(`${rightPad('Property:', this.maxPlacement - 9)}${fClone.identifier.name} ${fClone.card.toString()}`);
+                if (!fClone.identifier.name.includes('concept')) {
```

Comment:
I don't know the reason behind this change, but I do know that `concept` is a primitive, so technically it is not allowed to be a `Property` (it can only be a `Value`).  Is that why this change was made?

I also have a couple of other questions:
* Why are you checking using `includes('concept')` instead of `=== 'concept'`?
* If you're skipping `concept` identifiers here, are they just getting completely dropped or is there somewhere else they are being exported?

Reply:
These questions, of course, also apply to the code diffs below that are very similar.

Reply:
The idea here is that we wanted to manually remove CodeableConcept, Coding and Code from the CIMPL 6 exported files. Before they reach property however, they are converted to concept so this was implemented with that in mind.

Reply:
As for your question about includes, sometimes there may be a case where there are extra spaces present or other characters so that's why I was checking with includes and not ===.

Reply:
I thought we wanted to removed the `Element` declarations of `CodeableConcept`, `Coding`, and `Code` (e.g., remove the definition starting with `Element: CodeableConcept`, etc.)  If there are other elements that use them as _properties_ (not _values_), then this will drop it.  Are there other elements that use them as properties?  Or is it just that `CodeableConcept` has `Coding` as a property (e.g., the only uses are among themselves)?

Reply:
I don't understand how an `identifier.name` might end up with extra spaces or other characters.  I've never seen that before.  Do you know what causes that to happen?